ovs_gtp/README.md

```diff
@@ -0,0 +1,45 @@
+## Cleanup ovs build with Ansible.
+
+Goal: Remove the dependency on other Git repos by moving the build process for ovs to ansible and checkin the associated patches into a locally maintained subdir under magma/ovs_gtp
+
+In `roles/ovs_gtp`, you can find the main task in order to build the associate patches.
```

Comment:
Can we move the code to https://github.com/facebookincubator/magma/tree/master/lte/gateway/deploy/roles directory, so that we don't dilute the top level directory?